lib/challenge.js

```diff
@@ -48,9 +63,17 @@ var pet_owner;
   // Every pet can:
   // - make noise (each pet makes its own unique noise, as specified by `noise`.
 
-var some_pet;
+var some_pet {
 
 // Answer Starts Here
+Pet: cat,
```

Comment:
This will look for a variable `cat`, which I don't think is what you meant. If you wanted to have the value be text, it needs to be in a String (" ").